usermods/Tubes/Tubes.h

```diff
@@ -38,11 +38,16 @@ class TubesUsermod : public Usermod {
 
   public:
     void setup() {
-      pinMode(MASTER_PIN, INPUT_PULLUP);
-      pinMode(LEGACY_PIN, INPUT_PULLUP);
-      if (digitalRead(MASTER_PIN) == LOW) {
+
+      if (pinManager.isPinOk(MASTER_PIN)) {
```

Comment:
this is great - I assume we need to fix code in master.h too, where it reads button pins etc